docs/pages/careers/ROLE_TEMPLATE.md

```diff
@@ -82,6 +82,6 @@ You can find the other perks & benefits on the [careers](https://mui.com/careers
 
 Don't meet every requirement?
 Apply anyway!
-Research shows that certain folx are less likely to apply for a role than others [unless they meet 100%](https://hbr.org/2014/08/why-women-dont-apply-for-jobs-unless-theyre-100-qualified) of the outlined qualifications.
+Research shows that certain folks are less likely to apply for a role than others [unless they meet 100%](https://hbr.org/2014/08/why-women-dont-apply-for-jobs-unless-theyre-100-qualified) of the outlined qualifications.
```

Comment:
See https://www.reddit.com/r/AskLGBT/comments/xg16ap/folx_vs_folks/ for why. folx is a less formal version of folks, best to keep to the original form (I think an example of where less is more).